README.it.md

```diff
@@ -38,7 +57,7 @@ In alcuni dispositivi, devi anche abilitare [un'opzione aggiuntiva][control] per
 ### Sommario
 
  - Linux: `apt install scrcpy`
- - Windows: [download](README.md#windows)
+ - Windows: [download][direct-win64]
```

Comment:
No, please keep `README.md#windows`, so that an outdated translated README still points to the latest version to download.